cmd/cut-release/main.go

```diff
@@ -5,13 +5,16 @@ import (
 	"fmt"
 	"io/ioutil"
 	"log"
+	"os"
 	"os/exec"
 	"regexp"
 	"strings"
 
 	"github.com/plaid/go-envvar/envvar"
 )
 
+var functionDocsTemplate = "\n# Functions\n\n## loadMeshStreamingForURLAsync\n▸ **loadMeshStreamingWithURLAsync**(`url`: `string`): *Promise‹`void`›*\n\n*Defined in [index.ts:7](https://github.com/0xProject/0x-mesh/blob/%s/packages/browser-lite/src/index.ts#L7)*\n\nLoads the Wasm module that is provided by fetching a url.\n\n**Parameters:**\n\nName | Type | Description |\n------ | ------ | ------ |\n`url` | `string` | The URL to query for the Wasm binary |\n\n<hr />\n\n## loadMeshStreamingAsync\n\n▸ **loadMeshStreamingAsync**(`response`: `Response | Promise<Response>`): *Promise‹`void`›*\n\n*Defined in [index.ts:15](https://github.com/0xProject/0x-mesh/blob/%s/packages/browser-lite/src/index.ts#L15)*\n\nLoads the Wasm module that is provided by a response.\n\n**Parameters:**\n\nName | Type | Description |\n------ | ------ | ------ |\n`response` | `Response &#124; Promise<Response>` | The Wasm response that supplies the Wasm binary |\n\n<hr />"
```

Comment:
I think it would be cleaner to use a raw string literal (i.e., backticks) here? Then you could include literal line breaks instead of `/n`. You would have to escape the backticks in this case, but imo it will still be more readable overall.

Reply:
I thought about that, but there are a lot of backticks. I really wish that Golang would allow us to use backticks in raw strings (I'm really starting to dislike some of their language choices -- these issues provide some context on why you can't just escape backticks in raw strings https://github.com/golang/go/issues/18221 and https://github.com/golang/go/issues/24475). 

I'll change this and then we can decide which version is cleaner

Reply:
Forgot that you couldn't escape backticks inside a raw string literal without breaking it up into separate strings like this. I think this is easier to read but will likely be harder to change. I say we go back to the way it was before.